Exiled.API/Features/Player.cs

```diff
@@ -764,6 +764,14 @@ public bool BadgeHidden
             }
         }
 
+        /// <summary>
+        /// Gets a value indicating whether or not a player is in the pocket dimension.
+        /// </summary>
+        public bool IsInPocketDimension
+        {
+            get => ReferenceHub.transform.position.y < -1995f;
+        }
```

Comment:
Get the current room the player is in and check if the room type equals the pocket dimension type